SwinjectTests/BasicAssembly.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BasicAssembly.swift
+//  Swinject
+//
+//  Created by mike.owens on 12/9/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+import Swinject
+
+class AnimalAssembly: AssemblyType {
+    
+    func assemble(container: Container) {
+        container.register(AnimalType.self) { r in
+            return Cat(name: "Whiskers")
+        }
+    }
+}
+
+class FoodAssembly: AssemblyType {
+    
+    func assemble(container: Container) {
+        container.register(FoodType.self) { r in
+            return Sushi()
+        }
+    }
+}
+
+class PersonAssembly: AssemblyType {
+    
+    func assemble(container: Container) {
+        container.register(PetOwner.self) { r in
+            let definition = PetOwner()
+            definition.favoriteFood = r.resolve(FoodType.self)
+            definition.pet = r.resolve(AnimalType.self)
+            return definition
+        }
+    }
+}
+
+class PropertyAsssembly: AssemblyType {
+    func assemble(container: Container) {
+        container.register(AnimalType.self) { r in
+            return Cat(name: r.property("test.string")!)
+        }
+    }
+}
```

Comment:
Hound looks ignoring `excluded` settings in `.swiftlint.yml`. I excluded test code because unit tests using Quick usually get very long.

You can ignore the Hound warnings for unit tests though `BasicAssembly.swift` has no new line at end and `LoadAwareAssembly.swift` has two new lines at end. (Just small things. You can fix the two files if you care.)